lib/rules/method.js

```diff
@@ -71,6 +72,19 @@ function checkCallExpression(ruleHelper, callExpr, node) {
         ruleHelper.checkMethod(callExpr);
         break;
 
+
+    case "SequenceExpression":
+        // the return value of a SequenceExpression is the last expression.
+        // So, we create a new mock CallExpression with the actually called
+        // ... expression as the callee node and pass it to checkMethod()
+
+        const newCallExpr = Object.assign({}, callExpr);
+        const idx = node.expressions.length-1;
+        const called = node.expressions[idx];
+        newCallExpr.callee = called;
+        ruleHelper.checkMethod(newCallExpr);
+        break;
```

Comment:
@mozfreddyb at a first glance it seems that we could surround this case with a block, something like:
```
case "SequenceExpression": {
    ...
    const newCallExpr = ...
    ...
    ruleHelper.checkMethod(newCallExpr);
    break;
}
```

and avoid disabling the `no-case-declarations` eslint rule.